duckduckgo/macos-browser
# Add DuckPlayer help link

Task/Issue URL: https://app.asana.com/0/1207252092703676/1207452200346607/f

**Description**:
Adds a DuckPlayer help pages link to Settings

**Steps to test this PR**:
1. Go to Settings > Duck Player
2. Confirm you see a "Learn More" link below explanation
3. Clicking should open DuckPlayer help page in a new tab,.

![CleanShot 2024-09-16 at 16 04 49@2x](https://github.com/user-attachments/assets/ba50a179-1263-4c73-9d08-583aa4aa4119)
